README.md

```diff
@@ -10,6 +10,13 @@ UI components for Mapbox documentation projects.
 
 ## Installation
 
+**Requirements**
+
+- Node 14
```

Comment:
@katydecorah perhaps I missed it, but didn't see a place to upgrade Node in the `package.json`.